README.md

```diff
@@ -1,17 +1,22 @@
 # Amio
```

Comment:
Na zacatek bych dal odkazy na amio.io a dokumentaci, to je vzdycky opruz to hledat v tom readme